src/main/java/world/bentobox/bentobox/lists/Flags.java

```diff
@@ -430,6 +430,26 @@ private Flags() {}
      */
     public static final Flag ELYTRA = new Flag.Builder("ELYTRA",  Material.ELYTRA).type(Type.PROTECTION).defaultRank(RanksManager.VISITOR_RANK).listener(new ElytraListener()).build();
 
+    /*
+     * Basic tab flags - these set the other flags
+     */
+    public static final Flag BASIC_GATES_AND_DOORS = new Flag.Builder("BASIC_GATES_AND_DOORS", Material.IRON_TRAPDOOR).type(Type.BASIC)
```

Comment:
I prefixed all the basic flags with "BASIC" so that they would be obvious in the IDE and enum.